elbs/tf/common.sh

```diff
@@ -28,7 +28,7 @@ get_elb_access_group_id() {
 
 check_k8s_context() {
   current=$(kubectl config current-context)
-  if [ "${current}" != "${KOPS_NAME}" ]; then
+  if [[ ! "${KOPS_NAME}" =~ "${current}" ]]; then
```

Comment:
FYI I shorten my context names and use [bash functions of the same name](https://github.com/jgmize/dotfiles/blob/master/.bash_functions.d/k8s#L29-L33) to switch between clusters